tests/param_expand/01-basic.t

```diff
@@ -366,4 +366,57 @@ run_ok "${TEST_NAME_BASE}-18" cylc validate "suite.rc"
 cylc graph --reference 'suite.rc' >'18.graph'
 cmp_ok "${TEST_SOURCE_DIR}/${TEST_NAME_BASE}/18.graph.ref" '18.graph'
 
+# Reference by value, with -+ meta characters
+cat >'suite.rc' <<'__SUITE__'
+[cylc]
+    [[parameters]]
+        lang = c++, fortran-2008
+    [[parameter templates]]
+        lang = %(lang)s
+[scheduling]
+    [[dependencies]]
+        graph = "<lang=c++> => <lang = fortran-2008>"
```

Comment:
Many would argue against porting in this direction :grin:

Reply:
I didn't think about this one! Honest.